fn.go

```diff
@@ -118,15 +118,17 @@ func (f *Function) RunFunction(_ context.Context, req *fnv1beta1.RunFunctionRequ
 	}
 	log.Debug("Selected shell info", "shell", shell)
 	var stdout, stderr bytes.Buffer
+
 	cmd := exec.Command("/bin/"+shell, "-c", exportCmds+shellCmd)
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 
+	log.Debug("Running shell command")
 	cmderr := cmd.Run()
 	sout := strings.TrimSpace(stdout.String())
 	serr := strings.TrimSpace(stderr.String())
 
-	log.Debug(shellCmd, "stdout", sout, "stderr", serr)
+	log.Debug("Command done", "stdout", sout, "stderr", serr)
```

Comment:
Bearing in mind that the same container is used by all compositions implementing function-shell, and all invocations of function-shell from a composition, is this what we really want?

Reply:
I think so. Having the script dumped is not that useful, as you have the source anyway. And for long scripts, like mine, it makes the debug log unusable. The important part IMO is taht stdout and stderr get printed (and in case of an error, stdout contains script source anyway)